services/export/shared/csvExport.js

```diff
@@ -303,6 +303,9 @@ const downloadTransactionHistory = async ({ filename }) => {
 	const isFileExists = await staticFiles.exists(filename);
 	if (!isFileExists) throw new NotFoundException(`File ${filename} not found.`);
 
+	const isFile = await staticFiles.isFile(filename);
+	if (!isFile) throw new ValidationException(`${filename} is not a file.`);
```

Comment:
Could we also ensure that we return a `400 BAD REQUEST` response from the controller when we throw the ValidationException?